tests/integration/test_schema.py

```diff
@@ -2000,10 +2000,17 @@ async def test_schema_remains_constant(registry_async_client: Client) -> None:
     assert jsonlib.loads(res.json()["schema"]) == jsonlib.loads(schema_str)
 
 
-async def test_malformed_kafka_message(registry_async: KarapaceSchemaRegistry, registry_async_client: Client) -> None:
-    topic = registry_async.config["topic_name"]
+async def test_malformed_kafka_message(
+    command_line: Dict[str, str], registry_async: KarapaceSchemaRegistry, registry_async_client: Client
+) -> None:
+    if registry_async:
+        topic = registry_async.config["topic_name"]
+        bootstrap_servers = registry_async.config["bootstrap_uri"]
+    else:
+        topic = "_schemas"
+        bootstrap_servers = command_line["kafka_bootstrap_servers"]
```

Comment:
Please use the existing `kafka_servers` fixture instead. The fixture will properly wait for the servers to become available.